### ISSUE TITLE
Synthetic evaluation results

### ISSUE THREAD

**Issue by manu15sd** [DepthAnything/Depth-Anything-V2#212]
Hi, I read from other issues that the differences between the paper results and the report results are given by the different datasets (synthetic vs. real) used for training. It would be much appreciated if you could upload the results for the synthetic training just to compare and know if I'm getting the expected results. 

Thank you.

**Comment by LiheYoung** [comment]
Do you mean the results of solely trained on synthetic images? If so, you can refer to our Table 5.

**Comment by manu15sd** [comment]
I mean the fine-tuned metric version on synthetic (Hypersim and Virtual Kitty). Table 5 is not fine-tuned, isn't it?
